Tests/XcodeProjTests/Objects/Project/PBXProjEncoderTests.swift

```diff
@@ -419,6 +443,10 @@ class PBXProjEncoderTests: XCTestCase {
     private func loadTargetWithCustomBuildRulesProject() {
         proj = try! PBXProj(jsonDictionary: targetWithCustomBuildRulesDictionary().1)
     }
+
+    private func loadProjectWithXCLocalSwiftPackageReference() {
+        proj = try! PBXProj(jsonDictionary: iosProjectWithXCLocalSwiftPackageReference().1)
+    }
```

Comment:
nit: we can mark the method and test as throwing so the test fails rather than crashes in case of any issues, but I see other existing methods above follow the same pattern so can be a separate tidy up if need be

Reply:
Fixed, removed force unwraping 🤝